### ISSUE TITLE
Multus docker image version mismatch with the airgap package (`v1.27.3+rke2r1`)

### ISSUE THREAD

**Issue by sk4zuzu** [rancher/rke2#4480]
**Environmental Info:**
RKE2 Version: 
```
$ rke2 -v
rke2 version v1.27.3+rke2r1 (7c79e5e2c21c84613a301360512377bd9aebd833)
go version go1.20.5 X:boringcrypto
```

Node(s) CPU architecture, OS, and Version: 
```
$ uname -a
Linux k2a1 5.15.0-72-generic #79-Ubuntu SMP Wed Apr 19 08:22:18 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

Cluster Configuration:
3 servers, 3 nodes

**Describe the bug:**
Multus docker image version provided inside the airgap package is different from what the helm chart deploys by default.

```
root@k2a1:~# /var/lib/rancher/rke2/bin/crictl img | grep multus-cni
docker.io/rancher/hardened-multus-cni                                v3.9.3-build20230109                       395ff1b951853       419MB
```
```
root@k2a1:~# kubectl -n kube-system describe pod/rke2-multus-ds-gkkrc | grep 'Image:.*multus-cni'
    Image:         rancher/hardened-multus-cni:v3.9.3-build20230607
```

**Steps To Reproduce:**
1. Install RKE2 `v1.27.3+rke2r1`.
2. Use the [tarball method](https://docs.rke2.io/install/airgap#tarball-method), but also add the [multus package](https://github.com/rancher/rke2/releases/download/v1.27.3%2Brke2r1/rke2-images-multus.linux-amd64.tar.zst).
3. Compare what is in containerd and what the helm chart tries to deploy.

**Expected behavior:**
Multus deploys cleanly with exact docker image versions as advertised.

**Actual behavior:**
At least one image is not the same as in the package + in my case it enters a crash loop.

**Additional context / logs:**
Does not happen on `v1.27.2+rke2r1`.

**Comment by rbrtbnfgl** [comment]
Hi there was a misconfiguration between the airgapped images of multus and the one on the chart only on `v1.27.3` It should be fixed on `v1.27.4` that will be released this month. Could you use `v1.27.2` that is working and update to `v1.27.4` when it will be released?

**Comment by sk4zuzu** [comment]
> Hi there was a misconfiguration between the airgapped image of multus and the one on the chart only on `v1.27.3`. It should be fixed on `v1.27.4` that will be released this month. Could you use `v1.27.2` that is working and update to `v1.27.4` when it will be released?

Sure that's completely fine, thank you! :hugs:

**Comment by manuelbuil** [comment]
Fixed by https://github.com/rancher/rke2/issues/3786
